### PR TITLE
Ke smdba #7441

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/7441

@isbm, IIRC, your are the expert with smdba.  Can you please help with reviewing?  It's enough if you'd review the smdba related change!

### CODE REVIEW

File: adoc/bp_chap_suma_backup.adoc [hunk 1]
```diff
@@ -415,27 +431,25 @@ Start spacewalk-services with:
...
 
+[[config-smdb.recover-root]]
 == Recovering from a Crashed Root Partition
 
 
 This section provides guidance on restoring your server after its root partition has crashed.
-This section assumes you have setup your server similar to the procedure explained in Getting Started guide with separate partitions for the database and for channels mounted at [path]``/var/lib/pgsql``
- and [path]``/var/spacewalk/``
-.
+This section assumes you have setup your server similar to the procedure explained in the Getting Started guide with separate partitions for the database and for channels mounted at [path]``/var/lib/pgsql`` and [path]``/var/spacewalk/``.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: set up is two words as a verb

File: adoc/bp_chap_suma_backup.adoc [hunk 2]
```diff
@@ -415,27 +431,25 @@ Start spacewalk-services with:
...
 
 .Procedure: Recovering from a Crashed Root Partition
-. Start by installing SLES12 SP2 and the SUSE Manager Extension. Do not mount the [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions.
-. Once installation of SUSE Manager has completed shutdown services with [command]``spacewalk-service shutdown`` and the database with [command]``rcpostgresql stop``.
-. Mount your [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions and restore the directories listed in section one.
-. Start SUSE Manager services and the database with [command]``spacewalk-services start`` and [command]``rcpostgresql start``
-. SUSE Manager should now operate normally without loss of your database or synced channels.
+. Start by installing {sls}12 SPx and the SUSE Manager Extension. Do not mount the [path]``/var/spacewalk`` and [path]``/var/lib/pgsql`` partitions.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: Start the step with "Install ..."

File: adoc/bp_chap_suma_backup.adoc [hunk 3]
```diff
@@ -156,12 +175,12 @@ We recommend you run a manual database backup immediately after installation, or
...
 When [command]``smdba`` is run for the first time, or if you have changed the location of the backup, it will need to restart your database before performing the archive.
 This will result in a small amount of downtime.
-Note that regular database backups will not require any downtime.
+Regular database backups will not require any downtime.
 ====
 
 
 .Procedure: Performing a Manual Database Backup
-. Allocate permanent storage space for your backup. In this procedure, we will be using an NFS share located at [path]``/var/spacewalk/``.
+. Allocate permanent storage space for your backup. In this procedure, we will be using a directory located at [path]``/var/spacewalk/``.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: I'd remove the actor from this sentence (I'm cool with "you", less so with "we" ;)). "This example uses a directory ..."

File: adoc/bp_chap_suma_backup.adoc [hunk 4]
```diff
@@ -274,13 +299,12 @@ spacewalk-data-fsck
...
 
-In {susemgr}
-with an embedded database, archive logging is enabled by default.
+In {susemgr} with an embedded database, archive logging is enabled by default.
 This feature allows the database management tool [command]``smdba`` to perform hot backups.
 
 With archive log enabled, even more data is stored on the hard disk:
 
-* Postgresql maintains a limited number of archive logs. Using the default configuration, approx. 64 files with a size of 16 MiB are stored.
+* PostgreSQL maintains a limited number of archive logs. Using the default configuration, approx. 64 files with a size of 16 MiB are stored.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: New sentence, new line.

File: adoc/bp_chap_suma_backup.adoc [hunk 5]
```diff
@@ -291,7 +315,7 @@ Creating a user and syncing the channels:
 * SLE-Manager-Tools12-Updates-x86_64-SP2
 
 
-Postgresql will generate an additional ~1 GB of data.
+PostgreSQL will generate an additional ~1 GB of data.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: Ooh, I'm not sure about using a tilde to indicate "roughly". Can we reword this?

File: adoc/bp_chap_suma_backup.adoc [hunk 6]
```diff
@@ -348,7 +372,7 @@ For example if your database storage space is running low.
 The following procedure will guide you through moving the database to a new location for use by SUSE Manager.
 
 .Procedure: Moving the Database
-. The default storage location for {susemgr} is: [path]``/var/lib/pgsql/`` . You would like to move it, for example to: [path]``/storage/postgres/`` . To begin, stop the running database with:
+. The default storage location for {susemgr} is: [path]``/var/lib/pgsql/``. You would like to move it, for example to: [path]``/storage/postgres/``. To begin, stop the running database with:
```
[the remark this answers is not in the document]
Comment (keichwa) [reply adoc/bp_chap_suma_backup.adoc]: I'm going to propose something better or at least different.

File: adoc/bp_chap_suma_backup.adoc (hunk 3 above)
Comment (keichwa) [reply adoc/bp_chap_suma_backup.adoc]: yes, much better!

File: adoc/bp_chap_suma_backup.adoc [hunk 7]
```diff
@@ -304,7 +305,8 @@ This feature allows the database management tool [command]``smdba`` to perform h
 
 With archive log enabled, even more data is stored on the hard disk:
 
-* PostgreSQL maintains a limited number of archive logs. Using the default configuration, approx. 64 files with a size of 16 MiB are stored.
+* PostgreSQL maintains a limited number of archive logs.
+Using the default configuration, approx. 64 files with a size of 16 MiB are stored.
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: Oh, didn't notice this one last time: I would write out 'approximately'.

File: adoc/bp_chap_suma_backup.adoc [hunk 8]
```diff
@@ -372,14 +372,16 @@ For example if your database storage space is running low.
...
+
+. Stop the running database with:
 +
 
 ----
 # rcpostgresql stop
 ----
 +
-Shutdown running spacewalk services with:
+Shutdown the running spacewalk services with:
```
Comment (Loquacity) [inline adoc/bp_chap_suma_backup.adoc]: Shut down is two words as a verb.

**Comment by keichwa** [comment]
And now I'm not sure what we want to do with "spacewalk service(s)"...  The actual command is called "spacewalk-service" and then we talk about the name of the tool with "spacewalk services".  If the feeling that we should about the term "spacewalk services" and better use SUSE Manager services instead.  Or "{productname} services".

**Comment by Loquacity** [comment]
> And now I'm not sure what we want to do with "spacewalk service(s)"... The actual command is called "spacewalk-service" and then we talk about the name of the tool with "spacewalk services". If the feeling that we should about the term "spacewalk services" and better use SUSE Manager services instead. Or "{productname} services".

If I see "restart the spacewalk services(s)" then I would expect to be able to  issue:
`service spacewalk restart`

**Pull request merged**